paths_cli/tests/compiling/test_volumes.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+from unittest import mock
+from paths_cli.tests.compiling.utils import mock_compiler
+
+import yaml
+import openpathsampling as paths
+from openpathsampling.tests.test_helpers import make_1d_traj
+
+from paths_cli.compiling.volumes import *
+
+class TestBuildCVVolume:
+    def setup(self):
+        self.yml = "\n".join(["type: cv-volume", "cv: {func}",
+                              "lambda_min: 0", "lambda_max: 1"])
+
+        self.mock_cv = mock.Mock(return_value=0.5)
+        self.named_objs_dict = {
+            'foo': {'name': 'foo',
+                    'type': 'bar',
+                    'func': 'foo_func'}
+        }
+
+        self.func = {
+            'inline': "\n  ".join(["name: foo", "type: mdtraj"]),  # TODO
```

Comment:
why this todo? Leftover?
```suggestion
            'inline': "\n  ".join(["name: foo", "type: mdtraj"]),
```